src/renderers/shared/fiber/ReactFiberScheduler.js

```diff
@@ -911,7 +849,9 @@ module.exports = function<T, P, I, TI, PI, C, CX, PL>(
 
             // Complete the boundary as if it rendered null. This will unmount
             // the failed tree.
-            beginFailedWork(boundary.alternate, boundary, priorityLevel);
+            // TODO: We should unmount with task priority so that nothing else
```

Comment:
I think it can be a follow up along with the other error issues we know about. Not related to this PR, just something I noticed.